Cargo.toml

```diff
@@ -100,6 +101,7 @@ jsonrpsee = "0.20.0"
 keccak-hash = "0.10.0"
 lazy_static = "1.4.0"
 light-client = { path = "light-client" }
+openssl = { version = "0.10.66", features = ["vendored"]}
```

Comment:
The main downside of this solution is that `openssl` is a fairly large library, but I was unable to figure out an alternative solution (see other comment) ... but I'm open to any ideas

Reply:
the feature `native-tls-vendored` should be enabled for reqwest and `openssl = { version = "0.10.66", features = ["vendored"]}` should be removed I believe

Reply:
Nice, that seems to work fine locally!

Reply:
Could you try

```
reqwest = { version = "0.12.7", default-features = false, features = ["native-tls-vendored", "rustls-tls", "json"] }
```
changing this to

```
reqwest = { version = "0.12.7", default-features = false, features = ["rustls-tls-native-roots", "json"] }
```